diff --git a/node_modules/abstract-socket/binding.gyp b/node_modules/abstract-socket/binding.gyp
index 0beb060..03ea649 100644
--- a/node_modules/abstract-socket/binding.gyp
+++ b/node_modules/abstract-socket/binding.gyp
@@ -4,6 +4,8 @@
       'target_name': 'bindings',
       'defines': [ '_GNU_SOURCE=1' ],
       'sources': [ 'src/abstract_socket.cc' ],
+      'cflags': ['-std=c++17'],
+      'cflags_cc': ['-std=c++17'],
       'include_dirs': [
         '<!(node -e "require(\'nan\')")'
       ]
