diff --git a/node_modules/better-sqlite-helper/src/database.js b/node_modules/better-sqlite-helper/src/database.js
index 2972b5b..018a722 100644
--- a/node_modules/better-sqlite-helper/src/database.js
+++ b/node_modules/better-sqlite-helper/src/database.js
@@ -89,7 +89,8 @@ DB.prototype.connection = function () {
     mkdirp.sync(path.dirname(this.options.path))
     this.db = new Database(this.options.path, {
       readonly: this.options.readonly,
-      fileMustExist: this.options.fileMustExist
+      fileMustExist: this.options.fileMustExist,
+      verbose: this.options.verbose
     })
   } catch (e) {
     this.db = undefined
